src/Microsoft.IdentityModel.Protocols.OpenIdConnect/Microsoft.IdentityModel.Protocols.OpenIdConnect.csproj

```diff
@@ -15,6 +15,7 @@
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.IdentityModel.Protocols\Microsoft.IdentityModel.Protocols.csproj" />
     <ProjectReference Include="..\System.IdentityModel.Tokens.Jwt\System.IdentityModel.Tokens.Jwt.csproj" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
We should bump this up one level.
Inside <root>\build\dependencies.props there is a parameter <JsonNetVersion> use that for defining the Newtonsoft version in M.IM.Protocols.OpenIdConnect.csproj, M.IM.Tests.csproj, M.IM.Tokens.csproj, M.IM.Tokens.Tests.